.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - '12'
+  - 14.18.0
```

Comment:
Is this the right node version?

Reply:
We bless "active LTS releases": https://github.com/goodeggs/standards-and-best-practices/blob/2e4a4782bea91954abf3c3323c57f9dd2d8ca15f/modules-and-libraries/blessed.md#node. Right now that's 14 and 16.

(apparently 18 too starting literally today...)

Let's use 16?

Reply:
Sounds good.